hstool/templates/tool/indicators_add.html

```diff
@@ -22,6 +22,22 @@
       }
     });
   });
+
+  $(function() {
+    $('#datetimepicker1').datetimepicker({
```

Comment:
Why not use a class instead of two ids, and write the selector as: $('.datetimepicker')?